cmd/app.go

```diff
@@ -1280,6 +1282,23 @@ func askSubQuestion(q managementClient.SubQuestion) string {
 	return answer
 }
 
+func getAllDefaultAnswers(tv *managementClient.TemplateVersion, answers map[string]string) map[string]string {
```

Comment:
nit: this name sounds like you are setting all answers to default, when in reality you are filling in missing answers with the defaults. Maybe `fillInDefaultAnswers` ?